Eastern-Research-Group/mywaterway
# Feature/726 more climate tab changes

## Related Issues:
* [HMW-726](https://jira.epa.gov/browse/HMW-726)

## Main Changes:
* Removed fill from huc/county boundaries on extreme weather tab.
* Removed switch from drinking water systems.
* Fixed bug of weather events queries not using the layers definition expression (basically querying all weather events). This is what caused the weird issue where a location in Oklahoma had both an inland flood warning and a coastal flood warning.
  *  Note: There is a commit where I merged inland flooding and coastal flooding, but since I noticed and fixed this bug I reverted that change.
* Fixed issue of map not zooming to county on page refresh.

## Steps To Test:
1. Navigate to http://localhost:3000/community/180300030602/extreme-weather
2. Verify the Excessive heat switch turns on by default (it has a excessive heat watch)
3. Verify the map zooms to the county
4. Verify there is no fill on county or huc boundaries
5. Switch to a different tab
6. Verify the county/huc boundaries have fill again
7. Switch back to extreme weather
8. Verify the county/huc boundaries no longer have fill
9. Switch locations to verify other switches turn on when there are warnings, alerts, watches, etc.
10. Navigate to an inland location with a flood warning
11. Verify the coastal flooding row doesn't have the same warning as inland flooding row
12. Switch tabs ver
